BAZEL-JVM.md

```diff
@@ -495,19 +495,8 @@ Strive to make library targets as small as possible. In a situation where
 multiple Scala sources have no interdependencies you can use the
 `da_scala_library_suite` macro to automatically generate one library target per
 Scala source file, and bundle them in one target. This rule takes the same
-attributes as `da_scala_library` with the exception of
-`unused_dependency_checker_mode` which will always be disabled.
-
-If a Scala library defines macros then you must use the
-`da_scala_macro_library` rule instead of the above. Otherwise, you will encounter 
-compiler errors of the following form (formatted for readability):
-
-```
-error: macro annotation could not be expanded (the most common reason
-for that is that you need to enable the macro paradise plugin; another
-possibility is that you try to use macro annotation in the same
-compilation run that defines it)
-```
+attributes as `da_scala_library`, but enables `unused_dependency_checker_mode`
+by default.
```

Comment:
I don't think that's true. [`unused_dependency_checker_mode` is set to `False` within `rules_scala` itself](https://github.com/bazelbuild/rules_scala/blob/6c16cff213b76a4126bdc850956046da5db1daaa/scala/private/rules/scala_library.bzl#L118).
```suggestion
attributes as `da_scala_library` with the exception of
`unused_dependency_checker_mode` which will always be disabled.
```

Reply:
Aha, sorry, I misread it and thought it was about `da_scala_library`. I'll revert the text.

Reply:
Somehow I missed your suggestion and just did it manually. Not sure why. Anyway, done. 🙂

Reply:
Thank you!